### PR TITLE
Update Address.php

### PR DESCRIPTION
**Pull request by Machou**
Update to news french region

### CODE REVIEW

File: src/Faker/Provider/fr_FR/Address.php [hunk 1]
```diff
@@ -44,13 +44,26 @@ class Address extends \Faker\Provider\Address
     );
 
     private static $regions = array(
+        // Les treize régions métropolitaines, par ordre alphabétique de leurs noms et codes ISO 3166-2 :
```
Comment (ppelgrims) [inline src/Faker/Provider/fr_FR/Address.php]: Is there are source for this?

File: src/Faker/Provider/fr_FR/Address.php [hunk 2]
```diff
@@ -44,13 +44,26 @@ class Address extends \Faker\Provider\Address
...
+        'Pays de la Loire', 'Provence-Alpes-Côte d\'Azur', 
+
+        // Les cinq régions ultramarines, par ordre alphabétique des codes ISO 3166-2 :
+        'Guadeloupe', 'Guyane (française)', 'Martinique', 'La Réunion', 'Mayotte',
+
+        // Autres régions
+        'Nouvelle-Calédonie', 'Polynésie française', 'Saint-Barthélémy', 'Saint-Martin', 'Saint-Pierre-et-Miquelon',
+        'Wallis-et-Futuna'
+    );
+        /*
```
Comment (ppelgrims) [inline src/Faker/Provider/fr_FR/Address.php]: We're using git, so use `git diff` to see the difference between commits. You can safely remove the regions in the comments.

File: src/Faker/Provider/fr_FR/Address.php (hunk 1 above)
Comment (Machou) [reply src/Faker/Provider/fr_FR/Address.php]: New region in france in 2017
https://fr.wikipedia.org/wiki/R%C3%A9gion_fran%C3%A7aise

**Comment by pimjansen** [comment]
@Machou bump, could we resolve the comments so we can merge?

File: src/Faker/Provider/fr_FR/Address.php (hunk 2 above)
Comment (Machou) [reply src/Faker/Provider/fr_FR/Address.php]: Sorry, but I don't understand what the problem is?

**Comment by pimjansen** [comment]
I am going to close this PR as stale. If this still needs to be merged, please feel free to reopen it.
Thanks for contributing

**Pull request closed without merging**